clusterloader2/docs/design.md

```diff
@@ -246,6 +246,7 @@ struct TuningSet {
 	// Exactly one of the following should be set.
 	QpsLoad        *QpsLoad
 	RandomizedLoad *RandomizedLoad
+        PoissonLoad    *PoissonLoad
```

Comment:
Personally I wouldn't change the design doc. I think it's OK that it doesn't contain all changes made after the initial version of clusterloader2

Reply:
I believe that updating the design doc is also important to showcase all the features of clusterLoader2 - so that we can minimize the need to the user to look at the source code to understand clusterLoader2's features. What do you think?

Reply:
The goal of design doc is to document some decision process made when developing Clusterloader2 as of 2018.  As you can see it's not being updated since then: https://github.com/kubernetes/perf-tests/commits/master/clusterloader2/docs/design.md

I think what you mean here is a documentation, which is mostly missing in clusterloader.
The only place where we have some up to date state of Clusterloader is actually README.md: https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/README.md#measurement

You can add a section with tuningsets there and document changes (in this or in the next PR).

Reply:
I understand. I will add a section with tuningsets in the README.md in the next PR